Algorithms/GEMETD.py

```diff
@@ -0,0 +1,56 @@
+from Algorithms.ETDLB import ETDLB
+from Algorithms.BaseTD import BaseTD
+import numpy as np
+
+
+class GEMETD(ETDLB):
+    """
+    An ETD(0) implementation that uses GEM (aka GTD2(0) with x and x_p switched) to estimate emphasis.
+    """
+    def __init__(self, task, **kwargs):
+        super().__init__(task, **kwargs)
+        self.beta = self.task.GAMMA
+        self.gem_alpha = self.get('gem_alpha')  # Step size for GEM weights.
+        self.gem_eta = self.get('gem_eta')  # Regularization parameter for GEM; not needed for a fixed target policy.
```

Comment:
We need to read read the regularization parameter from the input, I believe. Example:

``self.gem_beta = kwargs['gem_beta']``
I chose beta for consistency. TDRC has a regularization parameter called TDRC_beta

We also need to add a parameter to learning.py. You don't need to worry about that. I'll take care of that myself.

Reply:
Gotcha! I misunderstood what self.get() was doing. I'll change `gem_eta` to `gem_beta` and update the code to read `gem_alpha` and `gem_beta` from the input.

Reply:
Thanks